content/en/real_user_monitoring/application_monitoring/browser/troubleshooting.md

```diff
@@ -112,14 +112,7 @@ The example code creates the following action event:
 }
 ```
 
-If an event or a request goes beyond any of the following limitations, it is rejected by the Datadog intake.
-
-| Property                                 | Limitation   |
-| ---------------------------------------- | ------------ |
-| Maximum number of attributes per event   | 256          |
-| Maximum attribute depth per event        | 20           |
-| Maximum event size                       | 256 KB       |
-| Maximum intake payload size              | 5 MB         |
+If an event or a request goes beyond any of limitations listed [here][], it is rejected by the Datadog intake.
```

Comment:
```suggestion
If an event or a request goes beyond any of the [listed RUM technical limitations][16], it is rejected by the Datadog intake.
```

Reply:
hi @MaelNamNam I think there was some confusion about where the technical limitations was published but I just disregarded that comment. however the link here was missing, and my suggestion was about making the text more clear than "here".